### PR TITLE
STORM-2295:KafkaSpoutStreamsNamedTopics changing the sequence of fields name while emitting data

### PR DESCRIPTION
**Pull request by pasalkarsachin1**
STORM-2295: I have just updated code to make sure sequence Fields should remains same even its added in set. In old implementation because of HashSet it was changing. Also added testcase for the same

### CODE REVIEW

**Comment by HeartSaVioR** [comment]
@pasalkarsachin1 
Please change the PR's title to same as issue's title or at least summary of this PR.

**Comment by pasalkarsachin1** [comment]
@HeartSaVioR  
Done. Thanks.

**Comment by revans2** [comment]
+1 the change looks good to me.  Great catch @pasalkarsachin1

**Comment by hmcl** [comment]
+1

**Comment by pasalkarsachin1** [comment]
Can someone merge it?

File: external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/KafkaSpoutStreamsNamedTopicsTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.apache.storm.kafka.spout;
```
Comment (harshach) [inline external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/KafkaSpoutStreamsNamedTopicsTest.java]: Please add the Apache License

**Comment by harshach** [comment]
@pasalkarsachin1 one of the files missing the License info. Please add that. We can merge it in after that.

**Comment by HeartSaVioR** [comment]
@pasalkarsachin1
And also please squash the commits into one after fixing license info. 
I recommend the title of commit message as `STORM-2295 KafkaSpoutStreamsNamedTopics changing the sequence of fields name while emitting data` and abstracted description to the body.

**Comment by hmcl** [comment]
I would suggest for title `STORM-2295 KafkaSpoutStreamsNamedTopics should return output fields with predictable ordering`

**Comment by pasalkarsachin1** [comment]
I squashed it.

**Comment by HeartSaVioR** [comment]
@pasalkarsachin1 
PR has been applied to master via ba406c16a27221dc4c1ca7ac3d3c3164cb669bfd and 1.x-branch via f4fdab3a45017dc042eda0c0c65d06a37f6cf904.
(I just cherry-picked 7168710 since it's squashed commit and others are not necessary.)

Could you please close this issue? Thanks in advance!
(Github doesn't recognize auto PR close message which PR is not target to master branch.)

**Pull request closed without merging**